modules/ROOT/pages/create-community.adoc

```diff
@@ -108,150 +108,39 @@ Configure the settings with one of these options:
 * Configure the member and guest user settings automatically using the default settings specified by the *Fix* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
 * Configure the member and guest user settings manually using custom settings applied in the following steps.
 
-=== Configure Member User Profile Permissions
+API Community Manager uses the following permission sets to define the visibility and access to external objects and Apex classes. 
```

Comment:
Do you need to say "the visibility and"? I would think that access implies visibility, but I don't know how the permission sets work.

If you leave in "visibility," the sentence needs some rewording because "visibility to external objects" (implied by the wording) is not grammatical. Maybe:

API Community Manager uses the following permission sets to define the visibility of external objects and Apex classes and access to these entities.